Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

dbcommands.c:770:1: error: conflicting types for 'dropdb'
dropdb(const char *dbname, bool missing_ok)
^
../../../src/include/commands/dbcommands.h:23:13: note: previous declaration is here
extern void dropdb(const char *dbname, bool missing_ok);
            ^
dbcommands.c:1367:1: error: conflicting types for 'AlterDatabase'
AlterDatabase(AlterDatabaseStmt *stmt, bool isTopLevel)
^
../../../src/include/commands/dbcommands.h:25:12: note: previous declaration is here
extern Oid      AlterDatabase(AlterDatabaseStmt *stmt, bool isTopLevel);
                ^
dbcommands.c:1977:1: error: conflicting types for 'get_database_oid'
get_database_oid(const char *dbname, bool missing_ok)
^
../../../src/include/commands/dbcommands.h:29:12: note: previous declaration is here
extern Oid      get_database_oid(const char *dbname, bool missingok);
                ^
indxpath.c:2917:1: error: conflicting types for 'relation_has_unique_index_for'
relation_has_unique_index_for(PlannerInfo *root, RelOptInfo *rel,
^
../../../../src/include/optimizer/paths.h:66:13: note: previous declaration is here
extern bool relation_has_unique_index_for(PlannerInfo *root, RelOptInfo *rel,
            ^
indxpath.c:3091:1: error: conflicting types for 'match_index_to_operand'
match_index_to_operand(Node *operand,
^
../../../../src/include/optimizer/paths.h:69:13: note: previous declaration is here
extern bool match_index_to_operand(Node *operand, int indexcol,
            ^
indxpath.c:3735:1: error: conflicting types for 'adjust_rowcompare_for_index'
adjust_rowcompare_for_index(RowCompareExpr *clause,
^
../../../../src/include/optimizer/paths.h:75:14: note: previous declaration is here
extern Expr *adjust_rowcompare_for_index(RowCompareExpr *clause,
             ^
regis.c:31:1: error: conflicting types for 'RS_isRegis'
RS_isRegis(const char *str)
^
../../../src/include/tsearch/dicts/regis.h:41:7: note: previous declaration is here
bool            RS_isRegis(const char *str);
                ^
regis.c:85:1: error: conflicting types for 'RS_compile'
RS_compile(Regis *r, bool issuffix, const char *str)
^
../../../src/include/tsearch/dicts/regis.h:43:7: note: previous declaration is here
void            RS_compile(Regis *r, bool issuffix, const char *str);
                ^
regis.c:218:1: error: conflicting types for 'RS_execute'
RS_execute(Regis *r, char *str)
^
../../../src/include/tsearch/dicts/regis.h:47:7: note: previous declaration is here
bool            RS_execute(Regis *r, char *str);
                ^
varlena.c:1845:1: error: conflicting types for 'varstr_sortsupport'
varstr_sortsupport(SortSupport ssup, Oid collid, bool bpchar)
^
../../../../src/include/utils/builtins.h:849:13: note: previous declaration is here
extern void varstr_sortsupport(SortSupport ssup, Oid collid, bool bpchar);
            ^
varlena.c:3264:1: error: conflicting types for 'SplitIdentifierString'
SplitIdentifierString(char *rawstring, char separator,
^
../../../../src/include/utils/builtins.h:859:13: note: previous declaration is here
extern bool SplitIdentifierString(char *rawstring, char separator,
            ^
varlena.c:3389:1: error: conflicting types for 'SplitDirectoriesString'
SplitDirectoriesString(char *rawstring, char separator,
^
../../../../src/include/utils/builtins.h:861:13: note: previous declaration is here
extern bool SplitDirectoriesString(char *rawstring, char separator,
            ^
varlena.c:3510:1: error: conflicting types for 'SplitGUCList'
SplitGUCList(char *rawstring, char separator,
^
../../../../src/include/utils/builtins.h:863:13: note: previous declaration is here
extern bool SplitGUCList(char *rawstring, char separator,
            ^
varlena.c:4022:1: error: conflicting types for 'replace_text_regexp'
replace_text_regexp(text *src_text, void *regexp,
^
../../../../src/include/utils/builtins.h:866:14: note: previous declaration is here
extern text *replace_text_regexp(text *src_text, void *regexp,
             ^
In file included from varlena.c:5586:
./levenshtein.c:73:1: error: conflicting types for 'varstr_levenshtein'
varstr_levenshtein(const char *source, int slen,
^
../../../../src/include/utils/builtins.h:850:12: note: previous declaration is here
extern int varstr_levenshtein(const char *source, int slen,
           ^
In file included from varlena.c:5588:
./levenshtein.c:68:1: error: conflicting types for 'varstr_levenshtein_less_equal'
varstr_levenshtein_less_equal(const char *source, int slen,
^
../../../../src/include/utils/builtins.h:854:12: note: previous declaration is here
extern int varstr_levenshtein_less_equal(const char *source, int slen,
           ^
guc.c:408:7: error: redefinition of 'log_duration' with a different type: 'bool' vs 'bool' (aka 'char')
bool            log_duration = false;
                ^
../../../../src/include/utils/guc.h:233:13: note: previous declaration is here
extern bool log_duration;
            ^
guc.c:409:7: error: redefinition of 'Debug_print_plan' with a different type: 'bool' vs 'bool' (aka 'char')
bool            Debug_print_plan = false;
                ^
../../../../src/include/utils/guc.h:234:13: note: previous declaration is here
extern bool Debug_print_plan;
            ^
guc.c:410:7: error: redefinition of 'Debug_print_parse' with a different type: 'bool' vs 'bool' (aka 'char')
bool            Debug_print_parse = false;
                ^
../../../../src/include/utils/guc.h:235:13: note: previous declaration is here
extern bool Debug_print_parse;
            ^
guc.c:411:7: error: redefinition of 'Debug_print_rewritten' with a different type: 'bool' vs 'bool' (aka 'char')
bool            Debug_print_rewritten = false;
                ^
../../../../src/include/utils/guc.h:236:13: note: previous declaration is here
extern bool Debug_print_rewritten;
            ^
guc.c:412:7: error: redefinition of 'Debug_pretty_print' with a different type: 'bool' vs 'bool' (aka 'char')
bool            Debug_pretty_print = true;
                ^
../../../../src/include/utils/guc.h:237:13: note: previous declaration is here
extern bool Debug_pretty_print;
            ^
guc.c:414:7: error: redefinition of 'log_parser_stats' with a different type: 'bool' vs 'bool' (aka 'char')
bool            log_parser_stats = false;
                ^
../../../../src/include/utils/guc.h:239:13: note: previous declaration is here
extern bool log_parser_stats;
            ^
guc.c:415:7: error: redefinition of 'log_planner_stats' with a different type: 'bool' vs 'bool' (aka 'char')
bool            log_planner_stats = false;
                ^
../../../../src/include/utils/guc.h:240:13: note: previous declaration is here
extern bool log_planner_stats;
            ^
guc.c:416:7: error: redefinition of 'log_executor_stats' with a different type: 'bool' vs 'bool' (aka 'char')
bool            log_executor_stats = false;
                ^
../../../../src/include/utils/guc.h:241:13: note: previous declaration is here
extern bool log_executor_stats;
            ^
guc.c:417:7: error: redefinition of 'log_statement_stats' with a different type: 'bool' vs 'bool' (aka 'char')
bool            log_statement_stats = false;            /* this is sort of all three
                ^
../../../../src/include/utils/guc.h:242:13: note: previous declaration is here
extern bool log_statement_stats;
            ^
guc.c:419:7: error: redefinition of 'log_btree_build_stats' with a different type: 'bool' vs 'bool' (aka 'char')
bool            log_btree_build_stats = false;
                ^
../../../../src/include/utils/guc.h:243:13: note: previous declaration is here
extern bool log_btree_build_stats;
            ^
guc.c:423:7: error: redefinition of 'check_function_bodies' with a different type: 'bool' vs 'bool' (aka 'char')
bool            check_function_bodies = true;
                ^
../../../../src/include/utils/guc.h:245:25: note: previous declaration is here
extern PGDLLIMPORT bool check_function_bodies;
                        ^
guc.c:424:7: error: redefinition of 'default_with_oids' with a different type: 'bool' vs 'bool' (aka 'char')
bool            default_with_oids = false;
                ^
../../../../src/include/utils/guc.h:246:13: note: previous declaration is here
extern bool default_with_oids;
            ^
guc.c:425:7: error: redefinition of 'SQL_inheritance' with a different type: 'bool' vs 'bool' (aka 'char')
bool            SQL_inheritance = true;
                ^
../../../../src/include/utils/guc.h:247:13: note: previous declaration is here
extern bool SQL_inheritance;
            ^
guc.c:428:7: error: redefinition of 'session_auth_is_superuser' with a different type: 'bool' vs 'bool' (aka 'char')
bool            session_auth_is_superuser;
                ^
../../../../src/include/utils/guc.h:248:13: note: previous declaration is here
extern bool     session_auth_is_superuser;
                ^
guc.c:4599:1: error: conflicting types for 'SelectConfigFiles'
SelectConfigFiles(const char *userDoption, const char *progname)
^
../../../../src/include/utils/guc.h:355:13: note: previous declaration is here
extern bool SelectConfigFiles(const char *userDoption, const char *progname);
            ^
guc.c:4989:1: error: conflicting types for 'AtEOXact_GUC'
AtEOXact_GUC(bool isCommit, int nestLevel)
^
../../../../src/include/utils/guc.h:359:13: note: previous declaration is here
extern void AtEOXact_GUC(bool isCommit, int nestLevel);
            ^
guc.c:5407:1: error: conflicting types for 'parse_int'
parse_int(const char *value, int *result, int flags, const char **hintmsg)
^
../../../../src/include/utils/guc.h:362:13: note: previous declaration is here
extern bool parse_int(const char *value, int *result, int flags,
            ^
guc.c:5493:1: error: conflicting types for 'parse_real'
parse_real(const char *value, double *result)
^
../../../../src/include/utils/guc.h:364:13: note: previous declaration is here
extern bool parse_real(const char *value, double *result);
            ^
guc.c:5554:1: error: conflicting types for 'config_enum_lookup_by_name'
config_enum_lookup_by_name(struct config_enum * record, const char *value,
^
../../../../src/include/utils/guc_tables.h:264:13: note: previous declaration is here
extern bool config_enum_lookup_by_name(struct config_enum * record,
            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]

--- src/backend/commands/dbcommands.c.orig	2020-08-10 21:21:12 UTC
+++ src/backend/commands/dbcommands.c
@@ -19,6 +19,7 @@
  */
 #include "postgres.h"
 
+#include <stdbool.h>
 #include <fcntl.h>
 #include <locale.h>
 #include <unistd.h>
--- src/backend/optimizer/path/indxpath.c.orig	2020-08-10 21:21:12 UTC
+++ src/backend/optimizer/path/indxpath.c
@@ -15,6 +15,7 @@
  */
 #include "postgres.h"
 
+#include <stdbool.h>
 #include <math.h>
 
 #include "access/stratnum.h"
--- src/backend/tsearch/regis.c.orig	2020-08-10 21:21:12 UTC
+++ src/backend/tsearch/regis.c
@@ -14,6 +14,8 @@
 
 #include "postgres.h"
 
+#include <stdbool.h>
+
 #include "tsearch/dicts/regis.h"
 #include "tsearch/ts_locale.h"
 
--- src/backend/utils/adt/levenshtein.c.orig	2020-08-10 21:21:12 UTC
+++ src/backend/utils/adt/levenshtein.c
@@ -23,6 +23,8 @@
  *
  *-------------------------------------------------------------------------
  */
+#include <stdbool.h>
+
 #define MAX_LEVENSHTEIN_STRLEN		255
 
 /*
--- src/backend/utils/adt/varlena.c.orig	2020-08-10 21:21:12 UTC
+++ src/backend/utils/adt/varlena.c
@@ -14,6 +14,7 @@
  */
 #include "postgres.h"
 
+#include <stdbool.h>
 #include <ctype.h>
 #include <limits.h>
 
--- src/backend/utils/misc/guc.c.orig	2020-08-10 21:21:12 UTC
+++ src/backend/utils/misc/guc.c
@@ -16,6 +16,7 @@
  */
 #include "postgres.h"
 
+#include <stdbool.h>
 #include <ctype.h>
 #include <float.h>
 #include <math.h>
